src/gridjs-vue.mjs

```diff
@@ -0,0 +1,246 @@
+import { Grid } from 'https://unpkg.com/gridjs/dist/gridjs.production.es.min.js'
```

Comment:
Can you please update Grid.js to v5? I have changed the module format a little bit which can affect this, see https://unpkg.com/browse/gridjs@5.0.1/

Reply:
Done